### ISSUE TITLE
`org-commits` without args could consider printing help

### ISSUE THREAD

**Issue by eseidelGoogle** [redonkulus/org-commits#6]
This is not important. :)  Just thought you'd like to know.

```
% org-commits
/usr/local/lib/node_modules/org-commits/node_modules/github/lib/index.js:359
                throw new Error("OAuth2 authentication requires a token or key & secret to be set");
                ^

Error: OAuth2 authentication requires a token or key & secret to be set
    at module.exports.__dirname.authenticate (/usr/local/lib/node_modules/org-commits/node_modules/github/lib/index.js:359:23)
    at OrgCommits.setupGithub (/usr/local/lib/node_modules/org-commits/dist/org-commits.js:119:25)
    at new OrgCommits (/usr/local/lib/node_modules/org-commits/dist/org-commits.js:77:14)
    at Object.<anonymous> (/usr/local/lib/node_modules/org-commits/bin/org-commits.js:19:18)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
    at Module.runMain (module.js:605:10)
```

It also does the same for any unknown arguments it seems? e.g.

```
%org-commits --version
/usr/local/lib/node_modules/org-commits/node_modules/github/lib/index.js:359
                throw new Error("OAuth2 authentication requires a token or key & secret to be set");
                ^

Error: OAuth2 authentication requires a token or key & secret to be set
    at module.exports.__dirname.authenticate (/usr/local/lib/node_modules/org-commits/node_modules/github/lib/index.js:359:23)
    at OrgCommits.setupGithub (/usr/local/lib/node_modules/org-commits/dist/org-commits.js:119:25)
    at new OrgCommits (/usr/local/lib/node_modules/org-commits/dist/org-commits.js:77:14)
    at Object.<anonymous> (/usr/local/lib/node_modules/org-commits/bin/org-commits.js:19:18)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
    at Module.runMain (module.js:605:10)
```

This is the newly released 1.2.1 FYI.

**Comment by eseidelGoogle** [comment]
If you have `GITHUB_ACCESS_TOKEN` set, you get a different message:
```
%org-commits
Missing required arguments: org
```

**Comment by redonkulus** [comment]
I'm taking a look now, the no arguments usecase is tricky because if you have a `.orgcommitsrc` file then you may not need to pass in any arguments to get it to work. I can make it smarter so that it could show the help if there is no `.orgcommitsrc` file and no arguments.

As for the `GITHUB_ACCESS_TOKEN`, I believe its because its going right to the setGitHub method which then gets that error. I recently switched from `optimist` to `yargs` so maybe how it handles required parameters is different. I will take a look. 

Thanks for the feedback!

**Comment by redonkulus** [comment]
@eseidelGoogle ok I've fixed this in v1.2.3. I was using an old version of `yargs` so updating that and using their `demandOption` param for `auth` will ensure the help menu shows up now.
